sql-db/narayana-transactions/src/test/java/io/quarkus/ts/transactions/MysqlTransactionGeneralUsageIT.java

```diff
@@ -15,9 +20,22 @@
 public class MysqlTransactionGeneralUsageIT extends TransactionCommons {
 
     static final int MYSQL_PORT = 3306;
-
-    @Container(image = "${mysql.57.image}", port = MYSQL_PORT, expectedLog = "port: " + MYSQL_PORT)
-    static final MySqlService database = new MySqlService();
+    @Container(image = "${mysql.80.image}", port = MYSQL_PORT, expectedLog = "port: " + MYSQL_PORT)
+    static final MySqlService database = new MySqlService().onPostStart(service -> {
+        // enable transactions recovery for user
+        var self = (MySqlService) service;
+        // RH MySQL image allow login root user locally without using password and ignoring set root password
```

Comment:
Doesn't that mean, that we can always add password and do not bother ourselves with the ternary on the next line?

Reply:
I tried it, and RH build seems to ignore the password value totally as it's stated in doc/description. So we need this or more like I add it if we in future run it with upstream MySQL image (now only RH image is used by default).

Description of RH MySQL image states:

> The root user has no password set by default, only allowing local connections. You can set it by setting the MYSQL_ROOT_PASSWORD environment variable. This will allow you to login to the root account remotely. Local connections will still not require a password.